patches/extensions-common-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/extensions/common/BUILD.gn b/extensions/common/BUILD.gn
+index 118c7255c77e74bd8c5cdb011c7f9bc96288de40..9e98323c80d1946639abc62ec14b978e521bc156 100644
+--- a/extensions/common/BUILD.gn
++++ b/extensions/common/BUILD.gn
+@@ -551,6 +551,8 @@ static_library("common") {
+     "//url",
+   ]
+ 
++  deps += ["//brave/components/skus/renderer"]
```

Comment:
cc: @brave/patch-reviewers

Reply:
Fixed - @bridiver found a better way to do the check. Basically, our subclassing of `ChromeExtensionsClient` has a method `IsScriptableURL`

cc: @diracdeltas @fmarier